common/containers/Tabs/Contracts/components/Interact/components/InteractForm/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React, { Component } from 'react';
+import translate from 'translations';
+import './InteractForm.scss';
+import { NetworkContract } from 'config/data';
+import { getNetworkContracts } from 'selectors/config';
+import { connect } from 'react-redux';
+import { AppState } from 'reducers';
+interface Props {
+  contracts: NetworkContract[];
+}
+
+interface State {
+  address: string;
+  abiJson: string;
+}
+class InteractForm extends Component<Props, State> {
+  public state = {
+    address: '',
+    abiJson: ''
+  };
+
+  public render() {
+    const { contracts } = this.props;
+    const { address, abiJson } = this.state;
+
+    let contractOptions;
+    if (contracts && contracts.length) {
+      contractOptions = [
+        {
+          name: 'Select a contract...',
+          value: null
+        }
+      ];
+
+      contractOptions = contractOptions.concat(
+        contracts.map(contract => {
+          return {
+            name: `${contract.name} (${contract.address.substr(0, 10)}...)`,
+            value: contract.address
+          };
+        })
+      );
+    } else {
+      contractOptions = [
+        {
+          name: 'No contracts available',
+          value: null
+        }
+      ];
+    }
+
+    // TODO: Use common components for address, abi json
+    return (
+      <div className="InteractForm">
+        <div className="InteractForm-address">
+          <label className="InteractForm-address-field form-group">
+            <h4>{translate('CONTRACT_Title')}</h4>
+            <input
+              placeholder="mewtopia.eth or 0x7cB57B5A97eAbe94205C07890BE4c1aD31E486A8"
+              name="address"
```

Comment:
`name="address"` is causing the input to autocomplete street addresses https://imgur.com/a/UmKbl, could we change the name attribute to `"contract_addtress"` instead?